packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -478,6 +488,20 @@ class EditableText extends StatefulWidget {
   final Color selectionColor;
 
   /// Optional delegate for building the text selection handles and toolbar.
+  ///
+  /// The [EditableText] widget used on its own will not trigger the display
```

Comment:
When used on its own, the [EditableText] widget will not display the selection toolbar.

Reply:
This might be over-simplified. When used on its own, the EditableText widget can display the selection toolbar. It'll just not trigger it by itself.

Reply:
Good point.